04-single-hop-6lowpan-icmp/example_test_guide.md

```diff
@@ -0,0 +1,168 @@
+_The following is a possible way to run the tests, this is just an example_
+## Goal: Check single hop 6lowpan icmp
+
+_note: Since the ping6 function is now async some interval times may need to
+be increased to compensate_
+
+### Pre-setup Steps
+1. Get 2 of the following boards: `samr21-xpro`, `iotlab-m3`
+2. Get the following boards:`remote-revb`, `arduino-zero` with `xbee` module
+2. Go into the `cd ${RIOTBASE}/tests/gnrc_udp` directory
+2. Setup a terminal for each board _(tmux is your friend)_
+3. Find the serial numbers and ports with `ls /dev/serial/by-id/`
+4. For the samr21 terminals use the following to setup the env
+_eg_ `export BOARD=samr21-xpro && export SERIAL=<SAMR_SERIAL> && export PORT=<SAMR_PORT>`
+
+`export BOARD=samr21-xpro && export SERIAL=ATML2127031800008334 && export PORT=/dev/ttyACM0`
+
+`export BOARD=samr21-xpro && export PORT_=ATML2127031800002161 && export PORT=/dev/ttyACM1`
+5. For the remote-revb:
+_eg_ `export BOARD=remote-revb && export PORT=<REMOTE_PORT> && export PORT_LINUX=${PORT}`
+
+`export BOARD=remote-revb && export PORT=/dev/ttyUSB0 && export PORT_LINUX=${PORT}`
+
+`export BOARD=remote-revb && export PORT=/dev/ttyUSB1 && export PORT_LINUX=${PORT}`
+6. For the iotlab-m3 the flashing must be done when the other is unplugged as
+there is no way to distinguish between nodes:
+_eg_ `export BOARD=iotlab-m3 && export PORT=<M3_PORT>`
+
+`export BOARD=iotlab-m3 && export PORT=/dev/ttyUSB3`
+
+`export BOARD=iotlab-m3 && export PORT=/dev/ttyUSB5`
+7. Use `make flash term` to access the nodes (or just `make term` if the
+firmware is already flashed)
+
+_note: Boards types can be changed to help vary the test coverage_
+
+Task #01 - ICMPv6 link-local echo with iotlab-m3
+================================================
+### Description
+
+ICMPv6 echo request/reply exchange between two iotlab-m3 nodes
+1. On the dest node use `ifconfig`
+2. Copy the link local address `inet6 addr: fe80::... scope: local VAL`, it
+should look like `fe80::1711:6b10:65f8:b43a`
+3. Ping from the src to the dest `ping6 -c 1000 -i 20 -s 0 <ll_addr>`
+4. Record the packet loss, it should be less then 10%
+_if packet loss is high make sure try to increase the -i value_
+5. check the packet buffer with `pktbuf`
+_as long as there are no weird characters it should be fine_
```

Comment:
What are "weird characters"? Better: "as long as there is no hex dump, the packet buffer is empty"

Reply:
Also: even if there is a hex dump, make sure to try again after a few seconds. Maybe it was just a packet that was stuck in a queue somewhere or some control message that you just received / are sending.

Reply:
Ahh I think I was just remembering what you said about as long as you don't see anything weird.  I will clarify that it is a hex dump and add the note about retry.